src/client/components/Forms/SignUp.js

```diff
@@ -1,54 +1,166 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import iconGithub from '../../assets/images/icons/github.svg';
+import './SignUp.style.css';
+import Button from '../Button/Button';
 
 export default function SignUp({ onSubmit }) {
+  const [name, setName] = useState('');
   const [email, setEmail] = useState('');
+  const [position, setPosition] = useState('');
+  const [image, setImage] = useState('');
+  const [linkedin, setLinkedin] = useState('');
+  const [github, setGithub] = useState('');
   const [password, setPassword] = useState('');
   const [passwordConfirm, setPasswordConfirm] = useState('');
+  const [validation, setValidation] = useState('');
   const handleEmailInput = (e) => setEmail(e.target.value);
+  const handleNameInput = (e) => setName(e.target.value);
+
+  const handlePositionInput = (e) => setPosition(e.target.value);
+  const handleImageInput = (e) => setImage(e.target.value);
+  const handleLinkedinInput = (e) => setLinkedin(e.target.value);
+  const handleGithubInput = (e) => setGithub(e.target.value);
   const handlePasswordInput = (e) => setPassword(e.target.value);
   const handlePasswordConfirmInput = (e) => setPasswordConfirm(e.target.value);
   const handleSubmit = (e) => {
     e.preventDefault();
+    if (password !== passwordConfirm) {
+      return setValidation('**Your password does not match**');
+    }
+    setValidation('');
     onSubmit({ email, password, passwordConfirm });
   };
   return (
-    <form onSubmit={handleSubmit}>
-      <label htmlFor="email">
-        Email:{' '}
-        <input
-          type="email"
-          name="email"
-          placeholder="Write your email"
-          value={email}
-          onChange={handleEmailInput}
-          required
-        />
-      </label>
-      <label htmlFor="password">
-        Password:{' '}
-        <input
-          type="password"
-          name="password"
-          placeholder="Write your password"
-          value={password}
-          onChange={handlePasswordInput}
-          required
-        />
-      </label>
-      <label htmlFor="password">
-        Password - confirm:{' '}
-        <input
-          type="password"
-          name="password"
-          placeholder="Write your password again"
-          value={passwordConfirm}
-          onChange={handlePasswordConfirmInput}
-          required
+    <div>
+      <form onSubmit={handleSubmit}>
+        <h2> Signup to Review Resume </h2>
+        {/* <button className="btn-Linkedin">Signup with LinkedIn</button>{' '} */}
+        <div className="button-forms">
+          <Button
+            buttonName="Signup With Linkedin"
+            style={{ backgroundColor: ' #0676f2' }}
+          ></Button>
```

Comment:
This component should be self-closing
```suggestion
          />
```